core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/options/IFramePlayerOptions.kt

```diff
@@ -34,6 +34,8 @@ class IFramePlayerOptions private constructor(private val playerOptions: JSONObj
             private const val MODEST_BRANDING = "modestbranding"
             private const val CC_LOAD_POLICY = "cc_load_policy"
             private const val CC_LANG_PREF = "cc_lang_pref"
+            private const val LIST = "list"
```

Comment:
can you explain what "list" and "listType" do?

Reply:
These are parameters of the YouTube player as described here: https://developers.google.com/youtube/player_parameters#Selecting_Content_to_Play
"listType" can either be "playlist" or "user_uploads", although "user_uploads" is kind of deprecated because it requires a YouTube username which only quite old channels seem to have.
"list" is either a playlist id (if "listtype" is set to "playlist") or the username (if "listtype" is set to "user_uploads").
Both ways work with this implementation.